docs/en/guides/serverless/oss/lambda-based-observability-adot.md

```diff
@@ -18,7 +18,7 @@ In this section of the observability best practices guide, we will deep dive on
 
 [AWS Distro for OpenTelemetry (ADOT)](https://aws-otel.github.io/docs/introduction) is a secure, production-ready, AWS-supported distribution of the Cloud Native Computing Foundation (CNCF) [OpenTelemetry (OTel)](https://opentelemetry.io/) project. Using ADOT, you can instrument your applications just once and send correlated metrics and traces to multiple monitoring solutions.
 
-AWS's managed [OpenTelemetry Lambda Layer](https://aws-otel.github.io/docs/getting-started/lambda) utilizes [OpenTelemetry Lambda Layer](https://github.com/open-telemetry/opentelemetry-lambda)  to export telemetry data asynchronously from AWS Lambda. It provides plug-and-play user experience by wrapping an AWS Lambda function, and by packaging the OpenTelemetry runtime specific SDK, trimmed down version of ADOT collector together with an out-of-the-box configuration for auto-instrumenting AWS Lambda functions. ADOT Lambda Layer collector components, such as Receivers, Exporters, and Extensions support integration with Amazon CloudWatch, Amazon OpenSearch Service, Amazon Managed Service for Prometheus, AWS X-Ray, and others. Find the complete list [here](https://github.com/aws-observability/aws-otel-lambda). ADOT also supports integrations with [partner solutions](https://aws.amazon.com/otel/partners).
+AWS's managed [OpenTelemetry Lambda Layer](https://aws-otel.github.io/docs/getting-started/lambda) utilizes [OpenTelemetry Lambda Layer](https://github.com/open-telemetry/opentelemetry-lambda) to export telemetry data. It provides plug-and-play user experience by wrapping an AWS Lambda function, and by packaging the OpenTelemetry runtime specific SDK, trimmed down version of ADOT collector together with an out-of-the-box configuration for auto-instrumenting AWS Lambda functions. ADOT Lambda Layer collector components, such as Receivers, Exporters, and Extensions support integration with Amazon CloudWatch, Amazon OpenSearch Service, Amazon Managed Service for Prometheus, AWS X-Ray, and others. Find the complete list [here](https://github.com/aws-observability/aws-otel-lambda). ADOT also supports integrations with [partner solutions](https://aws.amazon.com/otel/partners).
```

Comment:
Can we rephrase that to the following please?

[AWS Distro for OpenTelemetry Lambda](https://aws-otel.github.io/docs/getting-started/lambda) utilizes the upstream [OpenTelemetry Lambda Layers](https://github.com/open-telemetry/opentelemetry-lambda) to export telemetry data from AWS Lambda.